### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by devBustamento**
ready to get criticism, don't hold back!

### CODE REVIEW

File: OnboardingExperience/Program.cs [hunk 1]
```diff
@@ -6,9 +6,97 @@ class Program
...
+            }
+        }
+
+        static string AskQuestion(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+
+        public static int IntQuestion(string question, int length = 0) //has bug accepts 0, doesnt render 0
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: This comment seems to indicate that there's a bug?

File: OnboardingExperience/Program.cs [hunk 2]
```diff
@@ -6,9 +6,97 @@ class Program
...
+        {
+
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Unnecessary whitespace.

File: OnboardingExperience/Program.cs [hunk 3]
```diff
@@ -6,9 +6,97 @@ class Program
...
+            while (true)
+            {
+                var result = AskQuestion(question);//"the top"
+
+                if (!int.TryParse(result, out var num))//@Cody why does !int.TryParse(result, out int num)) not work?
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: It does

File: OnboardingExperience/Program.cs [hunk 4]
```diff
@@ -6,9 +6,97 @@ class Program
...
+                {
+                    Console.WriteLine($"You needed to type exactly a {length} digit number");
+                    continue;
+                }
+
+                return num;
+            }
+        }
+
+        public static bool YNbool(string question)
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: I'd change the name of this method. Not a very descriptive name.

File: OnboardingExperience/Program.cs [hunk 5]
```diff
@@ -6,9 +6,97 @@ class Program
...
+            bigKaBang(user);
+
+            Console.Write("\nPress any key to exit...");
+            Console.ReadKey(true);
+        }
+
+        private static void bigKaBang(User user) => Console.WriteLine($"{user.FirstName} {user.LastName}, at a hearty {user.Age}." +
+            $" \n bank account pre-existing: {user.AccOwn}, and your pin is set to {user.AccNo}");
+
+        private static bool IsOwnerQuestion(string question)
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: You should be calling your YNbool method, so you don't repeat yourself.

File: OnboardingExperience/User.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnboardingExperience
+{
+    class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int AccNo { get; set; }
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: This seems like an unnecessarily concise name.

File: OnboardingExperience/User.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+        public int Age { get; set; }
+        public bool AccOwn { get; set; }
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: `AccOwn` and `IsOwner` seems redundant.

File: OnboardingExperience/Program.cs [hunk 8]
```diff
@@ -6,9 +6,96 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var user = new User();
 
-            Console.ReadLine();
+            Console.WriteLine("Well well...\nWelcome to my Onbaording app, we need some info to get started...");
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Misspelled Onboarding, fyi. 😄

Comment (devBustamento) [reply OnboardingExperience/Program.cs]: nooooooooooooooo :'(

Comment (devBustamento) [reply OnboardingExperience/Program.cs]: certainly rip me apart with what else we can change though!

Comment (codytwinton) [reply OnboardingExperience/Program.cs]: @devBustamento We can do so at a later time, I need to review other students currently. Great job though.
